docs/integrations/persisting-store-data.md

```diff
@@ -735,15 +735,8 @@ export const useBearStore = create<MyState>()(
 
 ### How do I use it with Map and Set
 
-With the previous persist API, you would use `serialize`/`deserialize`
-to deal with `Map` and `Set` and convert them into
-an Array so they could be parsed into proper JSON.
-
-The new persist API has deprecated `serialize`/`deserialize`.
-
-Now, you will need to use the `storage` prop.
-Let's say your state uses `Map` to handle a list of `transactions`,
-then you can convert the Map into an Array in the storage prop:
+If your state uses `Map` or `Set` to handle a list of `transactions`,
+then you can convert the `Map` into an Array in the `storage` prop:
```

Comment:
I tried to make the simplest change here, but this may require a larger rewrite.

Reply:
This is for already a deprecated features, so can you open a new PR to the `main` branch to fix them?
I think all docs change can go directly to `main`, except for migration guide.

Reply:
This is addressed here: https://github.com/pmndrs/zustand/pull/2248